puppet/modules/web/manifests/vhost/stagingyum.pp

```diff
@@ -0,0 +1,81 @@
+# @summary Set up the yum vhost
+# @api private
+class web::vhost::stagingyum (
+  String[1] $stable,
```

Comment:
I'd argue we don't need none of the "stable", "latest" handling in here, as we always will refer with the "real" release versions to the repos